assets/src/carousel/index.js

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import Glider from 'glider-js';
+import 'glider-js/glider.css';
+
+/**
+ * WordPress dependencies
+ */
+import domReady from '@wordpress/dom-ready';
+
+domReady(() => {
+  const carouselWrappers = document.querySelectorAll(
+    '.web-stories-list__carousel'
+  );
+
+  const isRTL = 'rtl' === document.documentElement.getAttribute('dir');
```

Comment:
I think we should pass this down as a `isRTL` variable in a localize script, like we do in the editor. https://github.com/google/web-stories-wp/blob/1bfb6a2555baaf0eeac56bbaacc08cefa5c81341/includes/Story_Post_Type.php#L532